.github/workflows/pr-branch-checks.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/setup-node@v1
         name: setup node
         with:
-          node-version: '13.x'
+          node-version-file: '.nvmrc'
```

Comment:
wow, I did not know this feature existed, cool!

Reply:
So nice right?! It should be the default!